django/api/settings.py

```diff
@@ -198,7 +198,7 @@
     "name": "ITVR",
     "workers": 4,
     "timeout": 90,
-    "retry": 120,
+    "retry": 1260,
```

Comment:
is 21 minutes used here to retry after the 1200 second timeout in the scheduled job?

Reply:
So from the django_q docs, the retry time is "The number of seconds a broker will wait for a cluster to finish a task, before it’s presented again." Also: "The value must be bigger than the time it takes to complete longest task, i.e. [timeout](https://django-q.readthedocs.io/en/latest/configure.html#timeout) must be less than retry value and all tasks must complete in less time than the selected retry time."

Basically, the retry time has to be longer than the 1200 second timeout of the job because if, for example, the retry time is 600 seconds and the job takes 900 seconds, then at the 600 second mark (even though the job is still running and has not yet timed out), django_q will execute another instance of this job.